src/__tests__/render.test.tsx

```diff
@@ -191,11 +211,121 @@ test('debug changing component', () => {
   // eslint-disable-next-line no-console
   const mockCalls = (console.log as any as ConsoleLogMock).mock.calls;
 
-  expect(stripAnsi(mockCalls[4][0])).toMatchSnapshot(
+  expect(stripAnsi(mockCalls[0][0])).toMatchSnapshot(
```

Comment:
What caused the index change here?

Reply:
The fact that mocks are being cleared automatically do to the change I made in jest.config.js. We were checking the 4th call because it was called in previous test but now mock calls are reset before each test